lib/process-services-cloud/src/lib/process/process-header/components/process-header-cloud.component.spec.ts

```diff
@@ -225,8 +225,8 @@ describe('ProcessHeaderCloudComponent', () => {
             const lastModifiedElement = fixture.debugElement.query(By.css('[data-automation-id="header-lastModified"] .adf-property-value'));
 
             expect(component.dateFormat).toEqual('full');
-            expect(startedDateElement.nativeElement.innerText.trim()).toBe('Saturday, March 9, 2019 at 5:23:07 PM GMT+00:00');
-            expect(lastModifiedElement.nativeElement.innerText.trim()).toBe('Saturday, March 9, 2019 at 5:23:07 PM GMT+00:00');
+            expect(startedDateElement.nativeElement.innerText.trim()).toBe('Saturday, March 9, 2019 at 12:00:00 AM GMT+00:00');
```

Comment:
I had to change the expectation due to the zero time when dealing with date-only types